### PR TITLE
[14.0][MIG] web_notify: Migration to 14.0

### PR DESCRIPTION
**Pull request by kevinkhao**
Code reviewed + functional tests on user menu (has test buttons for notifications)

### CODE REVIEW

**Comment by tonygalmiche** [comment]
Does not work on runbot : 
image

File: web_notify/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Send instant notification messages to the user in live.
+
+This technical module allows you to send instant notification messages from the server to the user in live.
+Two kinds of notification are supported.
```
Comment (yostashiro) [inline web_notify/readme/DESCRIPTION.rst:4]: Not a question regarding the migration but what are these two kinds? Is it meant to be five listed below?

**Comment by kevinkhao** [comment]
@tonygalmiche strange, I tested in local and it works. Will investigate

File: web_notify/readme/DESCRIPTION.rst (hunk 1 above)
Comment (kevinkhao) [reply web_notify/readme/DESCRIPTION.rst:4]: Yes, that is an error. Fixed in migration commit.

**Comment by kevinkhao** [comment]
@tonygalmiche it doesn't work on runbot on the 12.0 version either. We have it on a 12.0 instance in production without problem.

**Comment by flotho** [comment]
Hi @kevinkhao , is this PR supposed to send desktop notifications ?

**Comment by kevinkhao** [comment]
@flotho No, it's for notifications like in the screenshot. Tested on v14 locally. Note that you need a working longpolling/bus for it to work.

Screenshot from 2020-11-13 09-39-56

**Comment by flotho** [comment]
Actually, I'm wondering what could be the use cases of this notifications without desktop notification.
How are you using this?
Your examples will help my understanding.

Regards

**Comment by kevinkhao** [comment]
@flotho you can check this module https://github.com/shopinvader/pattern-import-export/tree/12.0/pattern_import_export
the basic idea is that a user triggers a queue.job -> user is notified once the job is finished

**Comment by kevinkhao** [comment]
@flotho or maybe some button sends a request to an external API and it's slightly async, and you want to inform the user of if the request succeeded or not

**Comment by flotho** [comment]
ok, I see now.... thanks for the explanations

**Review by JunglistMNSQ — APPROVED** [review]

**Review by lmignon — APPROVED** [review]
Thank you @kevinkhao LGTM (Code review)

**Comment by ventor-dev** [comment]
@pedrobaeza , please, merge this PR. Seems it has all conditions to be merged

**Review by sbejaoui — APPROVED** [review]

**Review by hparfr — APPROVED** [review]

**Pull request merged**